xchange-lakebtc/pom.xml

```diff
@@ -25,6 +25,12 @@
 	<dependencies>
 
 		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-lang3</artifactId>
+			<version>3.3.2</version>
+		</dependency>
```

Comment:
Did you also remove the version line from `xchange-lakebtc/pom.xml`?

What jars are you trying to upload?  I don;t understand what you mean? Do you mean the Travis CI build is failing and you need help?